treeherder/perf/models.py

```diff
@@ -92,6 +92,8 @@ class PerformanceSignature(models.Model):
     ALERT_CHANGE_TYPES = ((ALERT_PCT, "percentage"), (ALERT_ABS, "absolute"))
 
     should_alert = models.BooleanField(null=True)
+    monitor = models.BooleanField(null=True)
```

Comment:
Does this mean that for monitored alerts we will have the following values: `monitor=True`, `should_alert=True`, `sheriffed=False`?